roofit/roofitcore/src/RooAbsData.cxx

```diff
@@ -2330,8 +2331,43 @@ Bool_t RooAbsData::hasFilledCache() const
 }
 
 ////////////////////////////////////////////////////////////////////////////////
+/// give access to internal data TTree (if it exists)
```

Comment:
Could you convert this into a sentence starting with a capital letter and describing a little better what this is supposed to do.